### ISSUE TITLE
Runtime errors

### ISSUE THREAD

**Issue by RootPathFinder** [arikogan/mysql-gelf#2]
All looks well, runs, grabs data, creates the status files with real data.. but on the second run I get:

[root@ovmm01 graylog]# sh mysql-status.sh 
Warning: Using a password on the command line interface can be insecure.
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
Runtime error (func=(main), adr=9): Divide by zero
/root/graylog

**Comment by arikogan** [comment]
Hi,
You are supposed to have two files,
`status.last`
`status.current`
Could you please run `grep Uptime <file>` on both files and paste the contents here?
Thanks.

**Comment by RootPathFinder** [comment]
It looks like current gets removed. At end of script the mv command does that.

On December 8, 2015 3:32:47 AM EST, Ariel Kogan notifications@github.com wrote:

> Hi,
> You are supposed to have two files,
> `status.last`
> `status.current`
> Could you please run `grep Uptime <file>` on both files and paste the
> contents here?
> Thanks.
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/arikogan/mysql-gelf/issues/2#issuecomment-162812899

## 

Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by arikogan** [comment]
You are right. Can you run the following at take a look if Update changes?
`watch -n 60 grep Uptime status.last`
Only thing I can think of for now is that files are not being changed and Uptime stays the same.

**Comment by RootPathFinder** [comment]
Appears to be updating.

Every 60.0s: grep Uptime status.last                                     

```
                                Wed Dec  9 15:58:08 2015
```

Uptime  517710
Uptime_since_flush_status   517710

**Comment by arikogan** [comment]
@hashfaster Dave, apologies I haven't been able to look into this. I need to change the script to allow some debugging but I can't find the time to do it now. Let's leave this open, hope to work on that soon.

**Comment by tetrode** [comment]
When running with cron, make sure that your path to mysql is added to the script
